.github/workflows/image.yaml

```diff
@@ -0,0 +1,36 @@
+name: Docker image
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  # Build image and deploy to ghcr.io
+  docker:
+    name: Build and deploy Docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE: ghcr.io/toothstone/cert-manager-webhook-anexia
```

Comment:
another `toothstone` - maybe use `$GITHUB_REPOSITORY` instead of `toothstone/cert-manager-webhook-anexia`?

Reply:
Good idea, although technically that is only by convention - ghcr pathes don't have to align with any repo.